client/src/RenderVideo.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import thumbUp from "../src/images/thumbUp.png";
+import thumbDown from "../src/images/thumbDown.png";
+
+
+const RenderVideo = ({ video, handleDeletedVideo }) => {
+    const [count, setCount] = useState(video.rating);
+
+    //Increasing Like Button
+    const increasingCount = () => {
+        setCount(count + 1);
+    };
+
+// Decreasing like button
+const decreasingCount = () => {
+    setCount(count -1);
+};
+
+return (
+  <section className="video">
+    <div>
+      <div className="title">
+        <h3>{video.title}</h3>
+        <iframe
+          width="500"
+          height="300"
+          src={`http://www.youtube.com/embed/${video.url.split("v=")[1]}`}
+          title="Video Player"
+          frameBorder="1"
```

Comment:
The `frameBorder` attribute of the `iframe` element is deprecated, so it is best not to use it in your code,